### DISCUSSION TITLE
How to update installation when deployed through yarn

### DISCUSSION THREAD

**Discussion by ehendrix23** [zwave-js/zwave-js-ui#1418] [Q&A]
My apologies if answered somewhere else but if so I could not find it.

I deployed zwavejs2mqtt as per the documentation in Quickstart using "compile your own code". I am running this on MacOS not in Docker. I first tried the packaged version but it seems that is not for MacOS as I always got:"exec format error" probably as this is for Linux and not MacOS.Thus I followed the steps for compiling my own code.

But what is the best method then to update? Following the same steps does not work as git clone then returns the directory is not empty.
Should I use git pull then for updating and then follow the further yarn steps? 

Appreciate the assistance.

**Comment by robertsLando** [comment]
```
git pull
yarn run build
// restart application
```
